firebase-vscode/src/workflow.ts

```diff
@@ -145,26 +145,32 @@ export function setupWorkflow(
 ) {
   extensionContext = context;
 
-  // Get user-defined VSCode settings.
-  const workspaceConfig = workspace.getConfiguration(
-    'firebase',
-    vscode.workspace.workspaceFolders[0].uri
-  );
-  const shouldDebug: boolean = workspaceConfig.get('debug');
+  var shouldDebug: boolean;
+  if (vscode.workspace.workspaceFolders) {
+    // Get user-defined VSCode settings.
+    const workspaceConfig = workspace.getConfiguration(
+      'firebase',
+      vscode.workspace.workspaceFolders[0].uri
+    );
+    shouldDebug = workspaceConfig.get('debug');
+  } else {
+    shouldDebug = false;
```

Comment:
Please just set the default value on initialization

Reply:
Done.